client/layout/masterbar/checkout.tsx

```diff
@@ -42,6 +53,7 @@ const CheckoutMasterbar: FunctionComponent< Props > = ( {
 	const [ isHelpCenterVisible, setIsHelpCenterVisible ] = useState( false );
 	const [ selectedArticle, setSelectedArticle ] = useState< Article | null >( null );
 	const [ footerContent, setFooterContent ] = useState( undefined );
+	const [ newItems, setNewItems ] = useState( false );
```

Comment:
Do we need local state here? Maybe directly use

```js
const newItems = data?.has_seen_whats_new_modal
```

Reply:
That was to prevent the initial moment, with a flickering of the dot. 
Because we need `const newItems = ! data?.has_seen_whats_new_modal`, to show the dot only when the user has not seen the modal. So in the inital moment, data is undefined and therefore newitems is true.

But I think that could be solved using `const newItems = ! isLoading && ! data?.has_seen_whats_new_modal;`

Reply:
Yes exactly!